mod/01_rating/set_pattern/01_metric_type.rb

```diff
@@ -0,0 +1,38 @@
+DEFAULT_METRIC_TYPE = 'Researched'
+
+@@options = {
+  junction_only: true,
+  index: 3,
+  anchor_parts_count: 1
+}
+
+def metric_type card_or_name
+  metric_name = card_or_name.is_a?(Card) ? card_or_name.name : card_or_name
+  mt_name = "#{metric_name}+*metric type"
+  mt_card = Card.fetch(mt_name, skip_modules: true, skip_type_lookup: true)
+  mt_card ||= card_or_name.is_a?(Card) && card_or_name.subfield(:metric_type)
+  mt_type = mt_card && mt_card.content.scan(/\[\[([^\]]+)\]\]/).flatten.first
+
+  mt_type || DEFAULT_METRIC_TYPE
+end
+
+def label name
```

Comment:
Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. You can also write as `label(*)` if you want the method to accept any arguments but don't care about them.